src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -22,26 +20,43 @@ namespace Altinn.Platform.Storage.Repository
     /// </summary>
     public class PgInstanceRepository: IInstanceRepository
     {
-        private static readonly string _deleteSql = "call storage.deleteinstance ($1)";
+        private static readonly string _deleteSql = "select * from storage.deleteinstance ($1)";
```

Comment:
The need for static fields are significantly reduced when there is only one instance of a class. `PgInstanceRepository` is added to DI as a Singleton.

Reply:
Changed